pkg/deployment/query.go

```diff
@@ -43,6 +43,13 @@ func (q *Query) Get() *appsv1.Deployment {
 	annotations := map[string]string{
 		"prometheus.io/scrape": "true",
 		"prometheus.io/port":   "16686",
+
+		// note that we are explicitly using a string here, not the value from `inject.Annotation`
+		// this has two reasons:
+		// 1) as it is, it would cause a circular dependency, so, we'd have to extract that constant to somewhere else
+		// 2) this specific string is part of the "public API" of the operator: we should not change
+		// it at will. So, we leave this configured just like any other application would
```

Comment:
I always prefer using a constant. Even we don't change it it's easier from the maintenance point of view.

Reply:
A local constant, instead of the same constant as the one used in the `inject` package? Could be, in that case, I'll consider it for #59.

Reply:
I mean a single constant unique for the project if possible

Reply:
Then no, I'll keep it as string following the reasoning on the comment